packages/macro/test/jsx-trans.ts

```diff
@@ -196,6 +196,50 @@ const cases: TestCase[] = [
         }} />;
       `,
   },
+  {
+    name: "JSX Macro inside JSX conditional expressions",
+    input: `
+       import { Trans } from '@lingui/macro'
+       ;<Trans>Hello, {props.world ? <Trans>world</Trans> : <Trans>guys</Trans>}</Trans>
+      `,
+    expected: `
+        import { Trans } from '@lingui/react'
+
+        ;<Trans
+            id={"Hello, {0}"}
+            values={{
+              0: props.world ? <Trans id={'world'} /> : <Trans id={'guys'} />
+            }}
+          />
```

Comment:
I'm thinking that we can automatically generate from 1 level ternary an ICU select expression: 

`Hello, { props.world, select, =true {world} =false {guys} }`

Or i'm overengineering...

Reply:
`Select` needs to always have the `other` case, so I don't really see a way how to do that.

Reply:
`Hello, { props.world, select, =true {world} other {guys} }`

But it's overengineering anyway